tools/online_editor/src/editor_widget.ts

```diff
@@ -131,6 +139,13 @@ class EditorPaneWidget extends Widget {
     this.editor_ready = this.setup_editor(this.contentNode);
   }
 
+  dispose() {
+    this.#disposables.forEach((d: monaco.IDisposable) => d.dispose());
+    this.#disposables = [];
+    this.#editor?.dispose();
+    this.#editor = null;
```

Comment:
It seems to me that the Widget base class' dispose function does a few things, so I think it should be called:

```suggestion
    this.#editor = null;
    super.dispose();
```

Reply:
True! Fixed.